src/pubtools/_marketplacesvm/tasks/community_push/command.py

```diff
@@ -36,8 +39,6 @@ class CommunityVMPush(MarketplacesVMPush, AwsRHSMClientService):
     def __init__(self, *args, **kwargs):
         """Initialize the CommunityVMPush instance."""
         self._rhsm_products: Optional[List[Dict[str, Any]]] = None
-        self._accts_dict: Dict[str, List[str]] = {}
-        self._snapshot_accts_dict: Dict[str, List[str]] = {}
```

Comment:
This was the main cause of the bug... Instead of returning the proper sharing accounts per mapping we were storing the first version of it globally, which was a mistake